deps/parser/parser.jl

```diff
@@ -0,0 +1,184 @@
+using JSON
+
+typestring = ""
+undotted_typestring = typestring[1:length(typestring)-1]
+
+type_translate_list = Dict(
+    "BigObject"          => typestring * "pm_perl_Object",
+    "OptionSet"          => typestring * "pm_perl_OptionSet",
+    "Matrix"             => typestring * "pm_Matrix{T} where T",
+    "Matrix<Rational>"   => typestring * "pm_Matrix{" * typestring * "pm_Rational}",
+    "Matrix<Integer>"    => typestring * "pm_Matrix{" * typestring * "pm_Integer}",
+    "Vector"             => typestring * "pm_Vector{T} where T",
+    "Vector<Rational>"   => typestring * "pm_Vector{" * typestring * "pm_Rational}",
+    "Vector<Integer>"    => typestring * "pm_Vector{" * typestring * "pm_Integer}",
+    "Integer"            => typestring * "pm_Integer",
+    "Rational"           => typestring * "pm_Rational",
+    "Set"                => typestring * "pm_Set",
+    "Anything"           => "Any"
+)
+
+## Dummy, will soon be generated
+filenames_list = Array{String,1}(
+    [ abspath( "tmp/polytope.json" ) ]
+)
+
+function julia_function_string(julia_name::String, polymake_name::String, app_name::String, julia_args::String, call_args::String, parameter_string::String )
+    return """
+function $julia_name( $julia_args )
+    $(typestring)application( "$app_name" )
+    return $(typestring)call_function( "$polymake_name$parameter_string", Array{Any,1}([ $call_args ]) )
+end
+
+"""
+end
+
+function julia_method_string(julia_name::String, polymake_name::String, app_name::String, julia_args::String, call_args::String, parameter_string::String )
+    return """
+function $julia_name( $julia_args )
+    $(typestring)application( "$app_name" )
+    return $(typestring)call_method( "$polymake_name$parameter_string", dispatch_obj, Array{Any,1}([ $call_args ]) )
```

Comment:
Is this `dispatch_obj` really always the same? Will this implicitly be in the `julia_args` list?

Reply:
I see this is implictly provided

Reply:
Yep, it is always the same. The methods are object methods.